docs/manual/java/releases/Migration14.md

```diff
@@ -52,6 +52,28 @@ cassandra-snapshot-store.keyspace = ${my-service.cassandra.keyspace}
 lagom.persistence.read-side.cassandra.keyspace = ${my-service.cassandra.keyspace}
 ```
 
+If you are using macOS or Linux, or have other access to a bash scripting environment, running this shell script in your base project directory can help add this configuration to existing services:
+
+```bash
+#!/bin/bash
+
+set -eu
+
+for svc_dir in *-impl; do
+    svc=${svc_dir%-impl}
+
+    # change $svc_dir to $svc on the next line to leave out the _impl suffix
+    keyspace=$(tr - _ <<<$svc_dir)
+    cat >> $svc_dir/src/main/resources/application.conf <<END
+$svc.cassandra.keyspace = $keyspace
```

Comment:
The `tr - _` command two lines up changes `-` to `_`.

It is still possible (but unlikely) that you could have other weird characters in the directory names that don't form valid keyspace names, but I think it would be easy to fix these manually on a case-by-case basis. I think it will fail at startup, so it will be easy to tell. I guess it's better to provide a solution that works most of the time than nothing at all.